README.md

```diff
@@ -97,6 +97,7 @@ _Web Analytics_
   * [Countly](https://count.ly) - Real time mobile & web analytics, crash reporting and push notifications platform. ([Source Code](https://github.com/countly)) `AGPLv3` `Javascript`
   * [Druid](http://druid.io/) - A distributed, column-oriented, real-time analytics data store. ([Source Code](https://github.com/druid-io/druid/)) `Apache` `Java`
   * [GoAccess](http://goaccess.io/) - Real-time web log analyzer and interactive viewer that runs in a terminal. ([Source Code](https://github.com/allinurl/goaccess)) `GPLv2` `C`
+  * [netdata](http://my-netdata.io/) - Real-time performance monitoring over the web, with plugins and alarms. ([Source Code](https://github.com/firehol/netdata)) `GPLv3+` `C`
```

Comment:
Looks good, only thing is the license might have to be changed to just GPLv3. (@nodiscc knows licenses best)